Core.lua

```diff
@@ -673,6 +673,66 @@ function R:IsAttemptAllowed(item)
 		return false
 	end
 
+	local _, _, _, _, _, _, _, _, _, LfgDungeonID = GetInstanceInfo()
+	local torghastLayerIDs = {
```

Comment:
are you referring to something like this?
![image](https://github.com/WowRarity/Rarity/assets/124485443/cb9e893e-1614-41f0-b69c-8d34923d90a3)

or do you want it to be more specific listing the torghast instance and layer = id

Reply:
I figured it could just be a mapping from "human readable name" to "LFG dungeon ID", same as the others.

Or what did you have in mind? [This DBC listing](https://wow.tools/dbc/?dbc=lfgdungeons&build=10.0.5.47660#page=1&search=torghast) should have all the IDs and names AFAICT.

Reply:
my interpretation was formatting it like everything else in sharedconstants but all the layer ids are just under 1 table, instead of having a unique name for each id

Reply:
See the other comments; both methods work although having just one table means we can't access the IDs directly.

But it really won't make a difference here, unless Blizzard adds new collectibles to Torghast. That, I think, is rather unlikely.